adapters/speechbrain/classification/model_adapter.py

```diff
@@ -0,0 +1,89 @@
+import os
+import torch
+import torchvision
+import torchaudio
+import dtlpy as dl
+import torch.nn.functional
+import torch.nn
+import logging
+from speechbrain.inference import EncoderClassifier
+import soundfile
```

Comment:
This is more for style than functionality: I think it looks better to have all the "imports" in a block and only later the "from ... import"s